packages/@aws-cdk/core/lib/cfn-fn.ts

```diff
@@ -22,6 +22,11 @@ export class Fn {
     return new FnRef(logicalName).toString();
   }
 
+  /** @internal */
+  public static _ref(logicalId: string): IResolvable {
+    return new FnRef(logicalId);
```

Comment:
You could get the same by using `Tokenization.reverseString(Fn.ref(...))`.

Will be slower, decide if you care :)

Reply:
That's not the same though, right? I need something that returns an `IResolvable`, which `Tokenization.reverseString()` doesn't.